src/App.tsx

```diff
@@ -1,18 +1,136 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Navigate, Route, Routes, useSearchParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { getPeople } from './api';
+import { Person } from './types';
+import { SortParams } from './components/types';
 
 export const App = () => {
+  const [loadingPeople, setLoadingPeople] = useState<boolean>(true);
+  const [error, setError] = useState<boolean>(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [visiblePeople, setVisiblePeople] = useState<Person[]>([]);
```

Comment:
it can be just a common variable.

Reply:
what do you mean?

Reply:
<img width="639" alt="image" src="https://github.com/user-attachments/assets/a94fa039-95bd-49cc-9e10-5cd498e1ab41">
actually yeah, you don't need a separate state for filtered people